src/core/middlewares/middlewares.go

```diff
@@ -74,6 +76,8 @@ func legacyAPISkipper(r *http.Request) bool {
 // MiddleWares returns global middlewares
 func MiddleWares() []beego.MiddleWare {
 	return []beego.MiddleWare{
+		beego.MiddleWare(session.Middleware()),
+		beego.MiddleWare(security.Middleware()),
 		csrf.Middleware(),
 		requestid.Middleware(),
 		log.Middleware(),
```

Comment:
Please help to move `requestid` and `log` in the front, thank you. 
The suggestion order is `requestid`, `log`, `session`, `csrf` and `security`.